client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -155,9 +158,16 @@ const useStatusChangeButton = (requisition: ResponseFragment) => {
             : '',
         });
 
+  const confirmationInfo = () => {
+    if (shipments?.totalCount === 0 && !isDisabled) {
+      return t('info.no-shipment');
+    }
+  };
```

Comment:
NIT: This looks good! Might be worth switching to a ternary for consistency with the other variables 😊. Just double-check the modal still works with undefined or an empty string as the false value.

Reply:
Thanks for the feedback - I did start with a ternary & empty string, which rendered an empty blue box!

I have updated this to a ternary with undefined :)